interactive/models.py

```diff
@@ -16,8 +18,8 @@
 from .notifications import send_welcome_email
 
 
-START_DATE = "2020-01-01"
-END_DATE = "2021-12-31"
+START_DATE = "2019-09-01"
+END_DATE = date.today().strftime("%Y-%m-%d")
```

Comment:
Do we want get the weds of the previous week here? To avoid any tails?

Reply:
Yes, that's a good point. I'll update it